Sources/FoundationEssentials/URL/URLTemplate_Expression.swift

```diff
@@ -152,61 +147,26 @@ extension URL.Template {
 
         let operatorRegex: Regex<(Substring, Substring?)>
         let separatorRegex: Regex<(Substring)>
-        let elementRegex: Regex<(Substring, Substring, Substring?, Substring??)>
-        let uriTemplateRegex: Regex<Regex<(Substring, Regex<OneOrMore<Substring>.RegexOutput>.RegexOutput)>.RegexOutput>
+        let elementRegex: Regex<(Substring, Substring, Substring?, Substring?)>
+        let uriTemplateRegex: Regex<(Substring, Substring)>
 
         private init() {
-            self.operatorRegex = Regex {
-                Optionally {
-                    Capture {
-                        One(.anyOf("+#./;?&"))
-                    }
-                }
-            }
+            self.operatorRegex = try! Regex(#"([\+#.\/;\?&])?"#)
```

Comment:
Do you know if it makes a difference whether you create a Regex this way or [the other one that specifies the output type](https://developer.apple.com/documentation/swift/regex/init(_:as:)-5z5nu)?

```swift
init(
    _ pattern: String,
    as outputType: Output.Type = Output.self
) throws
```
For example would it help the performance since we can bypass parsing the result from `AnyOutputRegex`?

Reply:
I'm ok with this as-is for now. We can always come back if we find this useful later.